capa/features/extractors/dnfile/helpers.py

```diff
@@ -187,22 +196,45 @@ def get_dotnet_managed_methods(pe: dnfile.dnPE) -> Iterator[DnType]:
             TypeNamespace (index into String heap)
             MethodList (index into MethodDef table; it marks the first of a continguous run of Methods owned by this Type)
     """
-    for row in iter_dotnet_table(pe, "TypeDef"):
-        for index in row.MethodList:
-            token = calculate_dotnet_token_value(index.table.number, index.row_index)
-            yield DnType(token, row.TypeName, namespace=row.TypeNamespace, member=index.row.Name)
+    for (rid, type_def) in iter_dotnet_table(pe, dnfile.mdtable.TypeDef.number):
+        type_def = cast(dnfile.mdtable.TypeDefRow, type_def)
+        for (idx, method) in enumerate(type_def.MethodList):
+            if method.table is None:
+                logger.debug("TypeDef[0x%X] MethodList[0x%X] table is None" % (rid, idx))
```

Comment:
```suggestion
                logger.debug("TypeDef[0x%X] MethodList[0x%X] table is None", rid, idx)
```

please lazily format logging messages